README.md

```diff
@@ -80,6 +80,9 @@ A community project to get an overview about the Galaxy project
 - [usegalaxy.* tools](https://github.com/galaxyproject/usegalaxy-tools)
 - [managing tools with Ephemeris](https://github.com/galaxyproject/ephemeris)
 
+#### Tool development
+- [LSP server](https://github.com/galaxyproject/galaxy-language-server/) Microsoft language server protocol for Galaxy Tool XML. Also works with [Jetbrains products](https://plugins.jetbrains.com/plugin/10209-lsp-support)
```

Comment:
Have you tested it with Jetbrains?

Reply:
At least that is what the plugin says it does

Reply:
Yeah, I was just wondering if you have tested it :)

Reply:
is there a running production instance of this server?

Reply:
No, I think you run those locally. Indeed VSC is starting the server for you.